src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
 // export const Person = ({ person }) => ();
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+
+    {person.age !== undefined && (
+      <p className="Person__age">I am {person.age}</p>
+    )}
+
+    {person.isMarried === true ? (
+      <p className="Person__partner">
+        {person.partnerName} is my {person.sex === 'm' ? 'wife' : 'husband'}
+      </p>
+    ) : (
+      <p className="Person__partner"> I am not married</p>
+    )}
```

Comment:
Let's do the check inside a p tag so that will have it only once and I would move it to a variable for better readability
```
const someVar = check here
<p>{someVar}</p>
```